From 23480f0de47aafe529fa1b7f97a769007382c45a Mon Sep 17 00:00:00 2001
From: Jon Lin <jon.lin@rock-chips.com>
Date: Mon, 14 Sep 2020 11:16:49 +0800
Subject: [PATCH] ARM: dts: rv1126-pinctrl.dtsi remove fspi cs1 request

In order to suppot the case probing both EMMC and SFC
with pinctrl enabled, we'd better to set GPIO0_D6 iomux
EMMC pins.

Change-Id: Ia9dccfff3acfae31c153222d191391dbc5e7ac12
Signed-off-by: Jon Lin <jon.lin@rock-chips.com>
---
 arch/arm/boot/dts/rv1126-pinctrl.dtsi | 2 --
 1 file changed, 2 deletions(-)

diff --git a/arch/arm/boot/dts/rv1126-pinctrl.dtsi b/arch/arm/boot/dts/rv1126-pinctrl.dtsi
index f99398ed9683..43c0e5c70dd3 100644
--- a/arch/arm/boot/dts/rv1126-pinctrl.dtsi
+++ b/arch/arm/boot/dts/rv1126-pinctrl.dtsi
@@ -303,8 +303,6 @@
 				<1 RK_PA3 3 &pcfg_pull_down>,
 				/* fspi_cs0n */
 				<0 RK_PD4 3 &pcfg_pull_up>,
-				/* fspi_cs1n */
-				<0 RK_PD1 3 &pcfg_pull_up>,
 				/* fspi_d0 */
 				<1 RK_PA0 3 &pcfg_pull_up>,
 				/* fspi_d1 */
-- 
2.35.3

